src/CIFAR100/CIFAR100.jl

```diff
@@ -21,10 +21,12 @@ module CIFAR100
         testdata,
 
         convert2image,
-        convert2features,
 
         download
 
+    @deprecate convert2features reshape
```

Comment:
Despite of a depwarn, is `CIFAR10.convert2features(CIFAR10.traintensor(Float32,1))` still working without error?

Reply:
no it will error, but it will but it will point in the right direction
```
julia> CIFAR10.convert2features(rand(32,32,3))
┌ Warning: `convert2features` is deprecated, use `reshape` instead.
│   caller = top-level scope at REPL[4]:1
└ @ Core REPL[4]:1
ERROR: DimensionMismatch("new dimensions () must be consistent with array size 3072")
Stacktrace:
 [1] (::Base.var"#throw_dmrsa#196")(::Tuple{}, ::Int64) at ./reshapedarray.jl:41
 [2] reshape at ./reshapedarray.jl:45 [inlined]
 [3] reshape at ./reshapedarray.jl:116 [inlined]
 [4] convert2features(::Array{Float64,3}) at ./deprecated.jl:43
 [5] top-level scope at REPL[4]:1
```
I think it's ok like this, it's not worth having a proper deprecation, people can figure this out (if anyone at all was using convert2features).

Reply:
I think the original design of this name was to introduce a conceptually symmetric name to `convert2image`, regardless of it's being trivial.

I can foresee a lot of example codes broken due to this... and non-power users would keep asking questions about how to fix things if this is totally removed.

Said that, if you think this as an edge usage, feel free to continue the deletion.

Reply:
I do think this is an edge case, with an easy fix pointed by the warning that should be within the reach of non-power users as well. It's not worth investing time in doing the deprecation properly (and I don't have much time these days). Maybe we can deprecate properly later if someone comes up with an issue for this